SPECS/libarrow/libarrow.spec

```
@@ -23,6 +23,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Source0:       https://github.com/apache/arrow/archive/refs/tags/apache-arrow-%{version}.tar.gz#/libarrow-%{version}.tar.gz
 Patch0001: 0001-python-pyproject.toml.patch
+Patch0002: CVE-2024-52338.patch
```

Just to align indentation. Usually, linter complains about this
```suggestion
Patch0001:       0001-python-pyproject.toml.patch
Patch0002:       CVE-2024-52338.patch
```